src/lib.rs

```diff
@@ -910,11 +910,33 @@ impl Executor {
     }
 
     /// Build all of the previously-defined dataflow blueprints and
-    /// then run them.
+    /// run them.
     ///
-    /// Will start up the number of workers specified on the command
-    /// line.
-    fn build_and_run(self_: Py<Self>, py: Python) -> PyResult<()> {
+    ///
+    /// # Arguments
+    ///
+    /// * `threads` - The number of per-process worker threads to use.
+    /// * `processes` - The number of processes that were started.
+    /// * `process` - Which process number this instance represents.
+    /// * `processes` - The total number of processes.
+    /// * `ctrlc` - A boolean flag to set a Ctrl-C handler on the main thread.
+    /// * `addresses` - An optional list of host/port addresses for other Bytewax workers.
+    #[args(
+        threads = "1",
+        processes = "0",
+        process = "0",
```

Comment:
bit of a nit but maybe `name` or `process_name`?  or `number_processes` below? they just look pretty similar offhand but nbd